filter/etc/e-smith/events/actions/nethserver-mail-filter-curl-actions

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/perl
+use NethServer::Password;
+use esmith::ConfigDB;
+my $db = esmith::ConfigDB->open_ro;
+my $password = NethServer::Password::store('rspamd');
+
+my $reject  = $db->get_prop('rspamd','SpamKillLevel') || '15';
+my $add     = $db->get_prop('rspamd','SpamTag2Level') || '6';
+my $rewrite = $add + 0.1;
+my $grey    = $db->get_prop('rspamd','SpamGreyLevel') || '4';
+my $status  = $db->get_prop('rspamd','SpamSubjectPrefixStatus') || 'disabled';
+
+if ($status eq 'disabled') {
+    $rewrite = 'null';
+}
+
+$error = qx(curl 'https://rspamd:$password\@127.0.0.1:980/rspamd/saveactions' --data '[$reject,$rewrite,$add,$grey]' -k);
```

Comment:
It's better to call the backend directly to avoid proxy error and HTTPS checks, something like:
```
curl 'http://rspamd:$password\@127.0.0.1:11334/rspamd/saveactions'
```
See also my doubts on using such API: https://community.nethserver.org/t/email-2-with-rspamd-1-7-x/9439/21?u=giacomo

Reply:
ok, for now we go to the disabled state, we need to document on the api before to go further.